gremlin-javascript/src/main/javascript/gremlin-javascript/lib/driver/driver-remote-connection.js

```diff
@@ -49,12 +49,14 @@ class DriverRemoteConnection extends RemoteConnection {
    * @param {String} [options.traversalSource] The traversal source. Defaults to: 'g'.
    * @param {GraphSONWriter} [options.writer] The writer to use.
    * @param {Authenticator} [options.authenticator] The authentication handler to use.
+   * @param {String} [options.headers] Add custom headers to the request.
```

Comment:
The correct type of the `@param` is `{Object}`.

What do you think about providing a little more detail, like: "An associative array containing the additional header key/values for the initial request"

Reply:
Hi, You're correct, i'm fixing this right away.
Thanks!

Reply:
Regarding other GLVs, yes i believe its important to have that.